src/model/frame-10.js

```diff
@@ -0,0 +1,82 @@
+"use strict";
+
+function Frame10() {
+  this._score1 = null;
+  this._score2 = null;
+  this._score3 = null;
+  this._total = null;
+}
+
+Frame10.prototype.addScore = function addScore( score ) {
+  if ( this._score1 === null ) {
+    this._score1 = score;
+  } else if ( this._score2 === null ) {
+    this._score2 = score;
+  } else {
+    this._score3 = score;
+  }
+};
+
+Frame10.prototype.isComplete = function isComplete() {
+  const firstBallStrike = this.score1 === 10;
+  const twoBallSpare = this.score1 + this.score2 === 10;
+  const needsThreeScores = firstBallStrike || twoBallSpare;
```

Comment:
I like how you have established when 3 scores needed for 10th frame. Nice simple and well thought through